packages/db/src/migrations/0024_create_interest_group.js

```diff
@@ -0,0 +1,16 @@
+import { sql } from "kysely"
+import { createTableWithDefaults } from "../utils.js"
+
+/** @param db {import('kysely').Kysely} */
+export async function up(db) {
+  await createTableWithDefaults("interest_group", { id: true, createdAt: true, updatedAt: true }, db.schema)
+    .addColumn("name", sql`text`, (col) => col.notNull())
+    .addColumn("description", sql`text`)
```

Comment:
What do you think about requirering description and link to be defined? i.e. not null?

Reply:
having a slack link isn't currently required by interest groups so i don't think that should be added, however i agree with adding description as not null

Reply:
fixtures are added and description is now not null 😊

Reply:
Nice! I'll take another look at the code.